stdlib/runtime-api/src/main/java/org/ballerinalang/stdlib/runtime/nativeimpl/GetInvocationContext.java

```diff
@@ -41,7 +41,7 @@
         return getInvocationContextRecord(Scheduler.getStrand());
     }
 
-    private static final String INVOCATION_CONTEXT_PROPERTY = "InvocationContext";
+    private static final String INVOCATION_CONTEXT_PROPERTY = "RuntimeInvocationContext";
```

Comment:
Shall we update as follows, in order to consistent with `AUTH_INVOCATION_CONTEXT_PROPERTY`?
```suggestion
    private static final String RUNTIME_INVOCATION_CONTEXT_PROPERTY = "RuntimeInvocationContext";
```